db/schema.rb

```diff
@@ -10,11 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_03_11_185140) do
+ActiveRecord::Schema.define(version: 2021_03_12_140240) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "books", force: :cascade do |t|
+    t.string "genre", null: false
+    t.string "author", null: false
+    t.string "image", null: false
```

Comment:
will this column contain the picture itself? or just a link to it?
If its just the link consider renaming it to `image_url` or `image_link`. If its the image itself, the string type won't work.

Reply:
I think it will contain a link, but I did it the way they asked me to do it. If you think so, I can do a rollback to the database and rewrite te migration so it will looks in the way you think it's better and also add "up" instead of "change".

Reply:
I did it